helix-core/src/main/java/org/apache/helix/controller/rebalancer/topology/Topology.java

```diff
@@ -55,16 +55,10 @@
   private final List<String> _liveInstances;
   private final Map<String, InstanceConfig> _instanceConfigMap;
   private final ClusterConfig _clusterConfig;
-  private final boolean _topologyAwareEnabled;
+  private final String _defaultPathPrefix = "Helix_default_";
```

Comment:
Please review the convention for defining constant Strings in Java.

https://stackoverflow.com/questions/9639007/defining-constant-string-in-java

With that said, is this prefix necessary?

Reply:
Updated.